Wikipedia/Code/ArticleLocationAuthorizationCollectionViewCell.swift

```diff
@@ -41,9 +41,6 @@ class ArticleLocationAuthorizationCollectionViewCell: ArticleLocationExploreColl
     }
 
     public func updateForLocationEnabled() {
-        guard WMFLocationManager.isAuthorized() else {
```

Comment:
This is superfluous, right? Just wanted to make sure I understood why it was removed.

Reply:
@mcleinman At this point yeah, I checked and the callers to this method now check their `locationManager`'s `isAuthorized` before calling this so I think it's safe. `isAuthorized` is no longer a class method by design, so it's more difficult to call from here. On their refactoring [write-up](https://industrial-binaries.co/case-study-refactoring-wikipedia-ios-app/) they go into their reasons in section 2.2: Class Methods.